Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -42,6 +43,9 @@ class NodeHelper implements ProtectedContextAwareInterface
     #[Flow\Inject]
     protected ContentRepositoryRegistry $contentRepositoryRegistry;
 
+    #[Flow\Inject]
+    protected NodeLabelGeneratorInterface $nodeLabelGenerator;
```

Comment:
Just wondering: Is the NodeLabelGenerator a global concept so it can't be used per content repository? I think that's what we (even _I_?) suggested. But just to make sure...

Reply:
Jup but we discussed that we are okay with keeping it this way at first. This pr doesnt change anything about the fundamentals, just how we access the label in fusion